phonebuddies/DatabaseConnector.py

```diff
@@ -83,7 +83,7 @@ def get_buddy_email_pairs(self):
             if not reject_pair:
                 valid_pairs.append(pair)
 
-        return pairs
+        return valid_pairs
```

Comment:
I think you fixed a bug here that I didn't even know existed 😅

Reply:
GPT caught it haha